src/services/dashboards/dashboard-detail/modules/DashboardDetailPreview.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+    <div ref="dashboardRef"
+         class="dashboard-detail-preview"
```

Comment:
It looks like `DashboardDetailPreview.vue` looks very similar with `DashboardDetailPage.vue`.
If there is some code changes on DetailPage, Should the same codes be written twice?

Reply:
That's right. If it's a code that should be displayed on the pdf, it should be written. 😢